load-test/laod_test.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+#Clean away old resources not useful anymore
```

Comment:
I'd suggest adding `set -e` to the script, which causes the whole script to stop executing and return failure when a line fails. This can help debugging and stop potentially undesired side-effects.
```suggestion
#!/bin/bash

set -e

#Clean away old resources not useful anymore
```

Reply:
I'll add also this